user/browserstack.md

```diff
@@ -15,17 +15,17 @@ and BrowserStack servers. Local Testing also has support for firewalls, proxies
 Once the secure connection is setup, all URLs work out of the box, including your webserver, local folders, as well as
 URLs with HTTPS.
 
-[local-testing]: https://www.browserstack.com/local-testing
+[local-testing]: https://www.browserstack.com/local-testing/?utm_source=rapise&utm_medium=partnered
 
-[local-binary]: https://www.browserstack.com/local-testing#command-line
+[local-binary]: https://www.browserstack.com/local-testing#command-line/?utm_source=rapise&utm_medium=partnered
```

Comment:
Hi @michal-at-travisci , could you please change the UTM link to this: https://www.browserstack.com/?utm_source=travis&utm_medium=partnered